packages/frontend/src/components/project/navigation/DesktopProjectNavigation.tsx

```diff
@@ -11,7 +12,7 @@ interface Project {
 }
 interface ProjectNavigationProps {
   project: Project
-  sections: ProjectDetailsSection[]
+  sections: (ScalingProjectDetailsSection | BridgeProjectDetailsSection)[]
```

Comment:
How about slightly shorter names?
```suggestion
  sections: (ScalingDetailsSection | BridgeDetailsSection)[]
```

Reply:
That's a good catch